lib/services/workflow-api-service.js

```diff
@@ -155,7 +236,22 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.findOne({
+            node: target,
+            _status: Constants.Task.ActiveStates
+        });
+    };
+
+    WorkflowApiService.prototype.getActiveWorkflows = function(query) {
+        return waterline.graphobjects.find(query);
+    };
+
+    WorkflowApiService.prototype.getWorkflowById = function(id) {
+        return waterline.graphobjects.needByIdentifier(id);
+    };
+
+    WorkflowApiService.prototype.destroyGraphDefinition = function(injectableName) {
```

Comment:
I agree with this.  As above, I think we should consider this a separate task that we track with a Github issue.